vocabularies/Org.OData.Validation.V1.xml

```diff
@@ -154,11 +154,14 @@ even though they are defined in an `Orders` entity set.</String>
           When applied to a collection-valued element, this annotation specifies the types allowed for members
           of the collection without mentioning the `Collection()` wrapper.
           The QualifiedTypeName may only include the `Collection()` wrapper
-          if the annotation is applied to an element with declared type `Edm.Untyped`.
-          It then allows a collection of instances of another type as the element's value.</String>

```

Comment:
I liked this additional clarification: why did you remove it?

Reply:
It's now in the type definition.